webdav/Dockerfile

```diff
@@ -0,0 +1,77 @@
+FROM httpd:latest
+
+# These variables are inherited from the httpd:alpine image:
+# ENV HTTPD_PREFIX /usr/local/apache2
+# WORKDIR "$HTTPD_PREFIX"
+
+# Copy in our configuration files.
+COPY conf/ conf/
+COPY hostcert_web1.key.pem /web1key.pem
+COPY hostcert_web1.pem /web1cert.pem
+COPY mod_want_digest/mod_want_digest.c /mod_want_digest.c
```

Comment:
Do we want to pull this file from Github, so that in case of updates from its maintainers, the container automatically has access to the latest version?

Reply:
It's a release that is almost 2 years old and the repository has been dormant since, I wouldn't pull from Github in this case.

Reply:
As discussed, let's stick with either a completely pinned dependency approach ( instead of the httpd:latest base image, use a pinned version) or a dynamic dependency approach ( pull the source from git). Intermixing these two approaches could lead to potentially unforseen container builds failing in the future.

Reply:
done.